client/src/components/Home.js

```diff
@@ -1,13 +1,12 @@
-import React from 'react'
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Bootstrap doesn't need to be imported into the individual components because it is already imported globally in index.scss